doc/jsk_2015_05_baxter_apc/apc2015_demo_sim.rst

```diff
@@ -18,7 +19,7 @@ Installation
   # setup repos
   cd ~/ros/jsk_apc2015_sim/src
   wstool init
-  wstool merge https://raw.githubusercontent.com/start-jsk/jsk_apc/0.1.3/jsk_2015_05_baxter_apc/simulation.rosinstall
+  wstool merge https://github.com/start-jsk/jsk_apc/blob/master/sim.rosinstall.kinetic
```

Comment:
I faced error here:
```
$ wstool merge https://github.com/start-jsk/jsk_apc/blob/master/sim.rosinstall.kinetic
ERROR in config: Invalid multiproject yaml format in [https://github.com/start-jsk/jsk_apc/blob/master/sim.rosinstall.kinetic]: mapping values are not allowed here
  in "<file>", line 115, column 73
```
I think you have to use `https://raw.githubusercontent.com`